### PR TITLE
Remove functional tests for opentelemetry instrumentation

### PR DESCRIPTION
**Pull request by ggainey**
(cherry picked from commit 981af2ef7b01d19b6b40b2c3c1a5dcbbc0bdf7a7)

### CODE REVIEW

File: pulpcore/tests/functional/api/test_status.py [hunk 1]
```diff
@@ -59,40 +59,24 @@
 
 
 @pytest.mark.parallel
-def test_get_authenticated(test_path, status_api_client, received_otel_span):
+def test_get_authenticated(test_path, pulpcore_bindings):
     """GET the status path with valid credentials.
 
     Verify the response with :meth:`verify_get_response`.
     """
     response = status_api_client.status_read()
```
Comment (gerrod3) [inline pulpcore/tests/functional/api/test_status.py:67]:
```suggestion
    response = pulpcore_bindings.StatusApi.status_read()
```

File: pulpcore/tests/functional/api/test_status.py [hunk 2]
```diff
@@ -59,40 +59,24 @@
...
 
 @pytest.mark.parallel
-def test_get_unauthenticated(test_path, status_api_client, anonymous_user, received_otel_span):
+def test_get_unauthenticated(test_path, pulpcore_bindings, anonymous_user):
     """GET the status path with no credentials.
 
     Verify the response with :meth:`verify_get_response`.
     """
     with anonymous_user:
         response = status_api_client.status_read()
```
Comment (gerrod3) [inline pulpcore/tests/functional/api/test_status.py:78]:
```suggestion
        response = pulpcore_bindings.StatusApi.status_read()
```

**Review by gerrod3 — CHANGES_REQUESTED** [review]

File: .github/workflows/scripts/install.sh
Comment (gerrod3) [inline .github/workflows/scripts/install.sh:1]: You are going to need to apply the plugin-template. The backport added settings not present in this branch's template-config.

Comment (ggainey) [reply .github/workflows/scripts/install.sh:1]: Thanks nice catch - I had *wondered* about that

**Review by gerrod3 — APPROVED** [review]

**Pull request merged**